estate/models/estate_property_type.py

```diff
@@ -0,0 +1,20 @@
+from odoo import fields, models
+
+class EstatePropertyType(models.Model):
+    _name = "estate.property.type"
+    _description = "Type of property (appartment, house...)"
+    _order = "name"
+
+    name = fields.Char(required=True)
+
+    _sql_constraints = [
+        ("unique_name", "UNIQUE(name)", "The name must be unique.")
+    ]
+
+    property_ids = fields.One2many("estate.property", "property_type_id", string="Properties with this type")
+    property_offer_ids = fields.One2many("estate.property.offer", "property_type_id", string="Offers on properties with this type")
+    property_offer_count = fields.Integer(compute="_compute_property_offer_count")
+
+    def _compute_property_offer_count(self):
+        for record in self:
+            record.property_offer_count =  record.property_offer_ids.__len__() or 0
```

Comment:
Keep it simple
```suggestion
            record.property_offer_count =  len(record.property_offer_ids)
```

Reply:
i think i had an error when i tried that, but maybe i hadnt `from builtins import len`

Reply:
[Always available builtins in Python](https://docs.python.org/3/library/functions.html)